### ISSUE TITLE
[FR] Ubuntu 22.04 Jammy Support

### ISSUE THREAD

**Issue by fujitatomoya** [Haivision/srt-prometheus-exporter#25]
#### Motivation

Currently Ubuntu 20.04 is only officially support, which has been 3 years already.
Adding support for Ubuntu 22.04 would be nice to have.

#### Feature description

Ubuntu 22.04 Jammy Support

#### Additional information

This needs to be done before https://github.com/Haivision/srt-prometheus-exporter/issues/11

**Comment by fujitatomoya** [comment]
closing in favor of https://github.com/Haivision/srt-prometheus-exporter/pull/27
